tests/acceptance/25_cf-execd/mailfilter_common.cf.sub

```diff
@@ -142,10 +142,10 @@ bundle agent compare_cf_execd_mail(test)
       "ok"               and => { "wanted_ok", "unwanted_ok" };
 
   reports:
-#      "unwanted_regex_list = $(unwanted_regex_list)";
-#      "wanted_regex = '$(wanted_regex)'";
-#      "unwanted_regex = '$(unwanted_regex)'";
-#      "output = '$(output)'";
+      "unwanted_regex_list = $(unwanted_regex_list)";
+      "wanted_regex = '$(wanted_regex)'";
+      "unwanted_regex = '$(unwanted_regex)'";
+      "output = '$(output)'";
```

Comment:
Will these be printed when the test fails in jenkins?

Reply:
@Lex-2008 please answer

Reply:
That's my expectation, yes.